src/test/java/io/jenkins/plugins/opentelemetry/init/StepExecutionInstrumentationInitializerTest.java

```diff
@@ -5,17 +5,26 @@
 
 package io.jenkins.plugins.opentelemetry.init;
 
-import io.opentelemetry.sdk.autoconfigure.spi.internal.DefaultConfigProperties;
-import org.junit.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
 
-import java.util.Collections;
+import hudson.ExtensionList;
+import java.util.List;
+import org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.ExecutorServiceAugmentor;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
 
 public class StepExecutionInstrumentationInitializerTest {
```

Comment:
I am not sure this test has any utility

Reply:
Please see https://github.com/jenkinsci/opentelemetry-plugin/pull/1139/commits/6d4f313549414ff0c4f99b39a51f8e699b5b3f41